server/tests/integration/graphql.test.js

```diff
@@ -346,6 +348,11 @@ describe('graphql', () => {
                                 locationOfData
                             }
                         }
+                        phaseChangeUpdate
+                        phaseTargetDate
+                        issues {
+                            __typename
+                        }
```

Comment:
Might be nice to figure out a way to populate these in tests, possibly through a mock service or something?

Reply:
Definitely! Perhaps we could discuss this further as well

Reply:
Yes good idea I have no idea right now how to do it lol